functions/fetch-yelp/index.js

```diff
@@ -2,8 +2,29 @@ const fetch = require('node-fetch');
 require('dotenv').config({ path: `.env.development.local` });
 
 const handler = async (event) => {
-  // add code here to fetch data from yelp API
-  // be sure to include the parameters from event.queryStringParameters
+  const zip = event.queryStringParameters.zip;
+  try {
+    const resp = await fetch(
+      `https://api.yelp.com/v3/businesses/search?categories=restaurants&location=${zip}`,
```

Comment:
Bringing in the `search` param here will allow you to pass the search term to the Yelp API and receive a more tailored response from the Yelp API. 

```js
https://api.yelp.com/v3/businesses/search?categories=restaurants&location=${zip}&term=${search}